stories/theme.AG_.introduction_agriculture/carousel_content.json

```diff
@@ -19,5 +19,9 @@
         "src":"https://www.youtube.com/embed/SPHtB88ra6c",
         "title":"Relative Wetness Root Zone Versus Groundwater Comparison",
         "caption":"These maps combine satellite and ground-based measurements to model the relative amount of water stored at two different depths: plant root level and underground. The brown regions represent dry conditions. The blue regions represent wet areas. The maps do not attempt to represent human consumption of water; but rather, they show changes in water storage related to weather, climate, and seasonal patterns. NASA researchers developed these maps with data incorporated data from the joint NASA-German Gravity Recovery and Climate Experiment (GRACE) and GRACE Follow-On (GRACE-FO) missions in partnership with the National Drought Mitigation Center."
+    },{
+        "src":"https://www.youtube.com/embed/6z58cOh_1TA",
```

Comment:
![image](https://github.com/NASA-IMPACT/veda-config-eic/assets/7830949/0541b745-9a76-412d-bcfa-d35132808e0f)

Reply:
uploaded all the videos, should be working now. (strangely it worked for me because I was logged in as the SVS page manager, even when embedded???)